pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1192,16 +1199,9 @@ func backupPlanHook(
 		jr := jobs.Record{
 			Description: description,
 			Username:    p.User(),
-			DescriptorIDs: func() (sqlDescIDs []descpb.ID) {
```

Comment:
I don't think we need to remove this in this change, so I'd leave it for now. We can decide later if we can stop setting this but there's no reason to couple it to the manifest-not-in-job change.

Reply:
Done.